src/theme-context-provider.tsx

```diff
@@ -82,6 +82,15 @@ const useDefaultHrefDecorator = () => {
     return (href: string) => href;
 };
 
+const defaultTextPresetsConfig: TextPresetsConfig = {
+    text5: {weight: 'light'},
+    text6: {weight: 'light'},
+    text7: {weight: 'light'},
+    text8: {weight: 'light'},
+    text9: {weight: 'light'},
+    text10: {weight: 'light'},
```

Comment:
shouldn't be "regular" the default?

Or maybe different text sizes should have different defaults